types/react-recaptcha-v3/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for react-recaptcha-v3 1.1
+// Project: https://github.com/codeep/react-recaptcha-v3
```

Comment:
So it closes: codeep/react-recaptcha-v3#19, doesn't it?

Reply:
actually yes!